engine/tests/mechanics/games/test_game_handlers.py

```
@@ -138,7 +139,10 @@ def test_moves_provisioned_when_ready(self, game_graph: Graph, game_block: GameB
         assert all(isinstance(action, Action) for action in actions)
         assert all(action.predecessor_id == game_block.uid for action in actions)
         assert all(action.successor_id == game_block.uid for action in actions)
-        assert all(action.payload == {"move": move} for action, move in zip(actions, ["win", "lose", "draw"]))
+        assert all(
+            action.payload == {"move": move}
+            for action, move in zip(actions, ["win", "lose", "draw"])
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)